workspace_visualization/src/capability_map_visual.cpp

```diff
@@ -0,0 +1,156 @@
+#include <OGRE/OgreVector3.h>
+#include <OGRE/OgreSceneNode.h>
+#include <OGRE/OgreSceneManager.h>
+
+#include <rviz/ogre_helpers/shape.h>
+#include <rviz/display_context.h>
+#include <rviz/display_factory.h>
+
+#include<tf2/LinearMath/Quaternion.h>
+
+#include "capability_map_visual.h"
```

Comment:
That's the problem I was dealing for so many days I have mentioned. Putting them in the include folder shows a lot of errors. However most of the visulization packages such as  
https://github.com/ros-visualization/visualization_tutorials/tree/groovy-devel/rviz_plugin_tutorials/src

https://github.com/wg-perception/object_recognition_ros_visualization/tree/master/src/rviz_plugin

the header files are in the same src folder